tests/test_integration.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+set -e
+
+# src_base="~/opt"
+src_base="$HOME/src/cran-server"
+
+package_tgz='httr_1.3.1.tar.gz'
+host='localhost'
+port='8081'
+
+url="http://$host:$port"
+
+cd $src_base
+
+# First install the app
+python setup.py install
+
+export FLASK_APP="$src_base/cranserver/server.py"
+export PYTHONPATH="$src_base/cranserver/"
+export CRANSERVER_FS_LOC="$HOME/cran"
+
+mkdir -p "$CRANSERVER_FS_LOC/src/contrib/"
+
+# spin up the server
+flask run -p "$port" &
```

Comment:
Need to make sure I spin down the server when done. E.g. on a second run you'll get a "port already in use" error.